
05.Jul 2020 17:15:49 predict module error
./2_RAW_DATA_DIR/sample_submission.csv not found.
Traceback (most recent call last):
  File ".\_3_predict.py", line 116, in predict
    submission_template = np.genfromtxt(''.join([local_script_settings['raw_data_path'], 'sample_submission.csv']),
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\npyio.py", line 1749, in genfromtxt
    fid = np.lib._datasource.open(fname, 'rt', encoding=encoding)
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\_datasource.py", line 195, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\_datasource.py", line 535, in open
    raise IOError("%s not found." % path)
OSError: ./2_RAW_DATA_DIR/sample_submission.csv not found.

05.Jul 2020 17:17:49 predict module error
./2_RAW_DATA_DIR/sample_submission.csv not found.
Traceback (most recent call last):
  File ".\_3_predict.py", line 116, in predict
    submission_template = np.genfromtxt(''.join([local_script_settings['raw_data_path'], 'sample_submission.csv']),
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\npyio.py", line 1749, in genfromtxt
    fid = np.lib._datasource.open(fname, 'rt', encoding=encoding)
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\_datasource.py", line 195, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\_datasource.py", line 535, in open
    raise IOError("%s not found." % path)
OSError: ./2_RAW_DATA_DIR/sample_submission.csv not found.

05.Jul 2020 17:18:41 predict module error
./2_RAW_DATA_DIR/sample_submission.csv not found.
Traceback (most recent call last):
  File ".\_3_predict.py", line 116, in predict
    submission_template = np.genfromtxt(''.join([local_script_settings['raw_data_path'], 'sample_submission.csv']),
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\npyio.py", line 1749, in genfromtxt
    fid = np.lib._datasource.open(fname, 'rt', encoding=encoding)
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\_datasource.py", line 195, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\_datasource.py", line 535, in open
    raise IOError("%s not found." % path)
OSError: ./2_RAW_DATA_DIR/sample_submission.csv not found.

05.Jul 2020 17:25:38 predict module error
./2_RAW_DATA_DIR/sample_submission.csv not found.
Traceback (most recent call last):
  File ".\_3_predict.py", line 116, in predict
    submission_template = np.genfromtxt(''.join([local_script_settings['raw_data_path'], 'sample_submission.csv']),
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\npyio.py", line 1749, in genfromtxt
    fid = np.lib._datasource.open(fname, 'rt', encoding=encoding)
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\_datasource.py", line 195, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\_datasource.py", line 535, in open
    raise IOError("%s not found." % path)
OSError: ./2_RAW_DATA_DIR/sample_submission.csv not found.

05.Jul 2020 17:27:06 predict module error
./2_RAW_DATA_DIR/sample_submission.csv not found.
Traceback (most recent call last):
  File ".\_3_predict.py", line 116, in predict
    submission_template = np.genfromtxt(''.join([local_script_settings['raw_data_path'], 'sample_submission.csv']),
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\npyio.py", line 1749, in genfromtxt
    fid = np.lib._datasource.open(fname, 'rt', encoding=encoding)
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\_datasource.py", line 195, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\_datasource.py", line 535, in open
    raise IOError("%s not found." % path)
OSError: ./2_RAW_DATA_DIR/sample_submission.csv not found.

05.Jul 2020 17:27:44 predict module error
./2_RAW_DATA_DIR/sample_submission.csv not found.
Traceback (most recent call last):
  File ".\_3_predict.py", line 116, in predict
    submission_template = np.genfromtxt(''.join([local_script_settings['raw_data_path'], 'sample_submission.csv']),
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\npyio.py", line 1749, in genfromtxt
    fid = np.lib._datasource.open(fname, 'rt', encoding=encoding)
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\_datasource.py", line 195, in open
    return ds.open(path, mode, encoding=encoding, newline=newline)
  File "C:\carpeta_p_pycharm\M5_kaggle_accuracy\venv\lib\site-packages\numpy\lib\_datasource.py", line 535, in open
    raise IOError("%s not found." % path)
OSError: ./2_RAW_DATA_DIR/sample_submission.csv not found.

05.Jul 2020 17:29:10 correct prediction process

05.Jul 2020 18:16:34 correct prediction process

05.Jul 2020 18:16:51 correct prediction process

05.Jul 2020 18:17:15 correct prediction process

05.Jul 2020 18:18:04 correct prediction process

05.Jul 2020 18:18:39 correct prediction process

07.Jul 2020 10:48:52 correct prediction process

07.Jul 2020 10:54:33 correct prediction process

07.Jul 2020 10:59:54 correct prediction process

07.Jul 2020 11:02:32 correct prediction process

07.Jul 2020 11:04:17 correct prediction process

07.Jul 2020 11:06:18 correct prediction process

07.Jul 2020 11:07:44 correct prediction process

07.Jul 2020 11:11:52 correct prediction process

07.Jul 2020 13:56:31 correct prediction process

07.Jul 2020 20:02:37 correct prediction process

07.Jul 2020 20:04:43 correct prediction process

07.Jul 2020 20:05:33 correct prediction process

07.Jul 2020 20:09:47 correct prediction process

07.Jul 2020 20:24:53 correct prediction process

07.Jul 2020 21:32:20 correct prediction process

07.Jul 2020 21:37:55 correct prediction process

07.Jul 2020 21:45:01 correct prediction process

07.Jul 2020 21:48:11 correct prediction process

07.Jul 2020 21:53:01 correct prediction process

07.Jul 2020 21:53:35 correct prediction process

07.Jul 2020 21:56:01 correct prediction process

07.Jul 2020 21:57:34 correct prediction process

07.Jul 2020 21:58:37 correct prediction process

07.Jul 2020 22:10:10 correct prediction process

07.Jul 2020 22:11:52 correct prediction process

07.Jul 2020 22:15:10 correct prediction process

07.Jul 2020 22:18:59 correct prediction process

07.Jul 2020 22:25:54 correct prediction process

07.Jul 2020 22:27:35 correct prediction process

07.Jul 2020 22:30:33 correct prediction process

07.Jul 2020 22:35:33 correct prediction process

07.Jul 2020 22:37:02 correct prediction process

07.Jul 2020 22:50:23 correct prediction process

07.Jul 2020 23:40:10 correct prediction process

07.Jul 2020 23:41:35 correct prediction process

07.Jul 2020 23:48:12 correct prediction process

07.Jul 2020 23:50:19 correct prediction process

07.Jul 2020 23:55:58 correct prediction process

08.Jul 2020 05:50:41 correct prediction process

08.Jul 2020 06:45:29 correct prediction process

08.Jul 2020 07:23:35 correct prediction process

08.Jul 2020 07:34:01 correct prediction process

08.Jul 2020 07:35:18 correct prediction process

08.Jul 2020 13:25:19 correct prediction process

08.Jul 2020 13:26:24 correct prediction process

08.Jul 2020 13:26:54 correct prediction process

08.Jul 2020 13:33:50 correct prediction process

08.Jul 2020 14:40:57 correct prediction process

08.Jul 2020 14:43:04 correct prediction process

08.Jul 2020 16:00:16 correct prediction process
